go/test/endtoend/prepare-statement/main_test.go

```diff
@@ -0,0 +1,272 @@
+}
+
+// DBInfo information about the database.
+type DBInfo struct {
+	Username     string
+	Password     string
+	Host         string
+	Port         uint
+	KeyspaceName string
+	Params       []string
+}
+
+func init() {
+	dbInfo.KeyspaceName = keyspaceName
+	dbInfo.Username = "testuser1"
+	dbInfo.Password = "testpassword1"
+	dbInfo.Params = []string{
+		"charset=utf8",
+		"parseTime=True",
+		"loc=Local",
+	}
+}
+
+var (
+	clusterInstance       *cluster.LocalProcessCluster
+	dbInfo                DBInfo
+	hostname              = "localhost"
+	keyspaceName          = "test_keyspace"
+	testingID             = 1
+	tableName             = "vt_prepare_stmt_test"
+	cell                  = "zone1"
+	mysqlAuthServerStatic = "/mysql_auth_server_static.json"
+	jsonExample           = `{
+		"quiz": {
+			"sport": {
+				"q1": {
+					"question": "Which one is correct team name in NBA?",
+					"options": [
+						"New York Bulls",
+						"Los Angeles Kings",
+						"Golden State Warriors",
+						"Huston Rocket"
+					],
+					"answer": "Huston Rocket"
+				}
+			},
+			"maths": {
+				"q1": {
+					"question": "5 + 7 = ?",
+					"options": [
+						"10",
+						"11",
+						"12",
+						"13"
+					],
+					"answer": "12"
+				},
+				"q2": {
+					"question": "12 - 8 = ?",
+					"options": [
+						"1",
+						"2",
+						"3",
+						"4"
+					],
+					"answer": "4"
+				}
+			}
+		}
+	}`
+	sqlSchema = `create table ` + tableName + ` (
+		id bigint auto_increment,
+		msg varchar(64),
+		keyspace_id bigint(20) unsigned NOT NULL,
+		tinyint_unsigned TINYINT,
+		bool_signed BOOL,
+		smallint_unsigned SMALLINT,
+		mediumint_unsigned MEDIUMINT,
+		int_unsigned INT,
+		float_unsigned FLOAT(10,2),
+		double_unsigned DOUBLE(16,2),
+		decimal_unsigned DECIMAL,
+		t_date DATE,
+		t_datetime DATETIME,
+		t_time TIME,
+		t_timestamp TIMESTAMP,
+		c8 bit(8) DEFAULT NULL,
+		c16 bit(16) DEFAULT NULL,
+		c24 bit(24) DEFAULT NULL,
+		c32 bit(32) DEFAULT NULL,
+		c40 bit(40) DEFAULT NULL,
+		c48 bit(48) DEFAULT NULL,
+		c56 bit(56) DEFAULT NULL,
+		c63 bit(63) DEFAULT NULL,
+		c64 bit(64) DEFAULT NULL,
+		json_col JSON,
+		text_col TEXT,
+		data longblob,
+		primary key (id)
+		) Engine=InnoDB`
+)
+
+func TestMain(m *testing.M) {
+	flag.Parse()
+
+	exitcode, err := func() (int, error) {
+		clusterInstance = cluster.NewCluster(cell, hostname)
+
+		defer clusterInstance.Teardown()
+
+		// Start topo server
+		if err := clusterInstance.StartTopo(); err != nil {
+			return 1, err
+		}
+
+		// create auth server config
+		SQLConfig := `{
+			"testuser1": {
+				"Password": "testpassword1",
+				"UserData": "vtgate client 1"
+			}
+		}`
+		if err := createConfig(mysqlAuthServerStatic, SQLConfig); err != nil {
+			return 1, err
+		}
+
+		// add extra arguments
+		clusterInstance.VtGateExtraArgs = []string{
+			"-mysql_auth_server_impl", "static",
+			"-mysql_server_query_timeout", "1s",
+			"-mysql_auth_server_static_file", clusterInstance.TmpDirectory + mysqlAuthServerStatic,
```

Comment:
I suggest removing the `/` from the value of mysqlAuthServerStatic and adding a `+ "/" +` here.